alternatives/dynamic-fps.mjs

```diff
@@ -6,10 +6,10 @@ export default new Mod(
 	"Improve performance when Minecraft is in the background.",
 )
 .icon("https://cdn.modrinth.com/data/LQ3K71Q1/icon.png")
-.add_version({ loader: ["fabric", "quilt"], v: [14, 15, 16, 17, 18, 19] })
+.add_version({ loader: ["quilt"], v: [19] }, { loader: ["fabric"], v: [14, 15, 16, 17, 18, 19] })
```

Comment:
```suggestion
.add_version({ loader: ["quilt"], v: [18, 19] }, { loader: ["fabric"], v: [14, 15, 16, 17, 18, 19] })
```

Same as before.
Even if it's not explicitly marked as supported, I believe they support it. Unless they do use unsupported features.